api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -12,18 +12,18 @@ import "gogoproto/gogo.proto";
 // Role-Based Access Control :ref:`configuration overview <config_http_filters_rbac>`.
 
 message RBAC {
-  // Specify the RBAC rules to be applied globally
-  config.rbac.v2alpha.RBAC rules = 1 [(validate.rules).message.required = true];
+  // Specify the enforcement mode RBAC rules to be applied globally.
+  // Disable the filter for enforcement mode if rules isn't set.
+  config.rbac.v2alpha.RBAC rules = 1;
+
+  // Specify the permissive mode RBAC rules to be applied globally.
+  // Disable the filter for permissive mode if permissive_rules isn't set.
+  config.rbac.v2alpha.RBAC permissive_rules = 2;
 }
 
 message RBACPerRoute {
-  oneof override {
-    option (validate.required) = true;
-
-    // Disable the filter for this particular vhost or route.
-    bool disabled = 1 [(validate.rules).bool.const = true];
```

Comment:
Why is this removed?

Reply:
as just mentioned in another comment,  this field could be inferred from whether rules is set(same as logic used in other places in this PR), need @rodaine confirm though.

Reply:
This was originally added as an explicit indicator that the filter is disabled for the vhost/route. Likewise, making it by-convention (ie, the `config.rbac.v2alpha.RBAC` being unset in the above config) permits the pathological configuration of RBAC where the global filter has no policies defined at all.